src/Parser/Shaman/Restoration/Modules/Talents/FlashFlood.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+
+import SpellLink from 'common/SpellLink';
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import { formatPercentage } from 'common/format';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import StatTracker from 'Parser/Core/Modules/StatTracker';
+import calculateEffectiveHealing from 'Parser/Core/calculateEffectiveHealing';
+
+import Abilities from '../Abilities';
+
+const BUFFER = 750;
```

Comment:
This seems like a way too big buffer, any reason it is so high? Having it at GCD cap could catch a lot of false positives at high haste.